build/linter/util/exclude_test.go

```diff
@@ -23,4 +23,5 @@ import (
 func TestShouldRun(t *testing.T) {
 	require.True(t, shouldRun("gofmt", "some.go"))
 	require.False(t, shouldRun("gofmt", "uca_generated.go"))
+	require.True(t, shouldRun("revive", "/pkg/meta/distributed_lock.go"))
```

Comment:
The PR that creates this file is not merged yet. #48971
```suggestion
	require.True(t, shouldRun("revive", "/pkg/meta/meta.go"))
```

Reply:
No problem. This test only uses the string, but not the file content.